src/common/ImageAvatar.js

```diff
@@ -1,11 +1,14 @@
 /* @flow */
 import React, { PureComponent } from 'react';
-import { ImageBackground, View } from 'react-native';
+import { ImageBackground, View, Text } from 'react-native';
 
 import type { ChildrenArray } from '../types';
 import { nullFunction } from '../nullObjects';
 import { Touchable } from './';
 
+import userImagePlaceholder from '../../static/img/user.png';
+import flagsReducers from '../chat/flagsReducers';
```

Comment:
`flagReducers` too

Reply:
This happened because of auto import, I'll just fix them.